connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/InboundIntermediateConnectorContextImpl.java

```diff
@@ -118,6 +119,9 @@ public void reportHealth(final Health health) {
     inboundContext.reportHealth(health);
   }
 
+  @Override
+  public void loadCancellationCallback(Consumer<Throwable> cancellationCallback) {}
```

Comment:
is it expected to do nothing?

Reply:
It is because InboundIntermediate did not have a cancellation mechanism, it wraps another context which is supposed to contain the cancel mechanism. We could speak about this one over a meeting as it is a bit complicated